action/protocol/poll/util.go

```diff
@@ -318,9 +318,9 @@ func setCurrentBlockMeta(
 
 // allBlockMetasFromDB returns all latest block meta structs
 func allBlockMetasFromDB(sr protocol.StateReader, blocksInEpoch uint64) ([]*BlockMeta, error) {
-	keys := [][]byte{blockMetaKey(math.MaxUint64, blocksInEpoch)}
+	keys := make([][]byte, blocksInEpoch)
```

Comment:
`math.MaxUint64` corresponds to max key in old code (using `Filter`)
now switch to use `keys`, it is not needed